tendrl/commons/jobs/__init__.py

```diff
@@ -126,14 +126,21 @@ def _run(self):
                             Event(
                                 ExceptionMessage(
                                     priority="error",
-                                    job_id=job.job_id,
-                                    flow_id = the_flow.parameters['flow_id'],
                                     publisher=NS.publisher_id,
                                     payload={"message": "error",
                                              "exception": e
                                              }
                                 )
                             )
+                            Event(
+                                Message(
+                                    job_id=job.job_id,
+                                    flow_id = the_flow.parameters['flow_id'],
+                                    priority="error",
+                                    publisher=NS.publisher_id,
+                                    payload={"message": "Job failed"}
```

Comment:
How is "Job failed" explaining anything to the user? Add the e.msg from exception caught